tests/snapshots/tests.contrib.httpx.test_httpx.test_configure_service_name_env.json

```diff
@@ -0,0 +1,182 @@
+[[
```

Comment:
this fixture is not the same as the original snap file.

the original snap only has 1 trace and 1 span, this one has 6 (?)

Reply:
Resolved by https://github.com/DataDog/dd-apm-test-agent/pull/30